.github/workflows/ci.yml

```diff
@@ -228,12 +229,18 @@ jobs:
           # gnupg2 is for signing the resulting binary.
           install: |
             apt-get update -q -y
-            apt-get install -q -y build-essential libz-dev git python3-pip gnupg2 curl
+            apt-get install -q -y build-essential libz-dev git python3-pip gnupg2 curl zip
+            touch /tmp/github_env
           run: |
             set -e
             export PATH="/utils:$PATH"
             git config --global --add safe.directory "$(pwd)"
             ./mill -i copyDefaultLauncher artifacts/
+            ./mill -i ci.setShouldPublish
+            if grep -q "SHOULD_PUBLISH=true" /tmp/github_env; then
+              .github/scripts/generate-os-packages.sh
+            fi
+      - run: sudo chown -R $USER .
```

Comment:
Just to make sure I chown all files after the vm ran, so we don't have to worry about access denied errors etc.